CMakeLists.txt

```diff
@@ -30,6 +39,11 @@ catkin_package(
     DEPENDS Boost Eigen OMPL OpenRAVE
 )
 
+if(OR_OMPL_HAS_BOOSTSMARTPTRS)
+else(OR_OMPL_HAS_BOOSTSMARTPTRS)
+add_definitions(-std=c++11)
```

Comment:
Fixed! And the `config.h` file was already being generated, so I had just added the new variable to it.